README.md

```diff
@@ -138,6 +138,35 @@ Returns route map details
     }
     ```
 
+  **Trip Fare**
+    ---
+    Returns trip fare details
+    * **URL**
+
+      `http://bmtcmob.hostg.in/api/tripfare/details`
+    * **Method**
+
+      `POST`
+    * **Data Params**
+
+      ```javascript
+    {
+        'adults' : <noOfAdults>,
+        'destination' : <destination>,
+        'serviceType' : <serviceType>,
+        'source' : <source>
+      }
+      ```
+      Service types can be either
+  * ordinary
```

Comment:
actually there arent CamelCase .. they arent variable either <actually forgot to add spaces> ... any way snake_case is here